### PR TITLE
Display a warning when calling "copyFiles" with an invalid path

### PR DESCRIPTION
**Pull request by Lyrkan**
This PR adds a warning when `copyFiles()` is called with a `from` value that isn't an existing directory.

This is kind of related to #445 but even if the friendly-errors-webpack-plugin worked properly in this case the origin of the error would probably not be obvious (it'd most likely contain a reference to the temporary entry).

### CODE REVIEW

File: lib/config-generator.js [hunk 1]
```diff
@@ -148,11 +148,25 @@ class ConfigGenerator {
...
 
-        if (this.webpackConfig.copyFilesConfigs.length > 0) {
+        const copyFilesConfigs = this.webpackConfig.copyFilesConfigs.filter(entry => {
+            const copyFrom = path.resolve(
+                this.webpackConfig.getContext(),
+                entry.from
+            );
+
+            if (!fs.existsSync(copyFrom) || !fs.lstatSync(copyFrom).isDirectory()) {
+                logger.warning(`The "from" option of copyFiles() should be set to an existing directory but "${entry.from}" does not seem to be one. No file will be copied for this entry.`);
```
Comment (weaverryan) [inline lib/config-generator.js]: "entry" might not be the best word here... because it means something else

File: lib/config-generator.js [hunk 2]
```diff
@@ -148,11 +148,25 @@ class ConfigGenerator {
...
         }
 
-        if (this.webpackConfig.copyFilesConfigs.length > 0) {
+        const copyFilesConfigs = this.webpackConfig.copyFilesConfigs.filter(entry => {
+            const copyFrom = path.resolve(
+                this.webpackConfig.getContext(),
+                entry.from
+            );
+
+            if (!fs.existsSync(copyFrom) || !fs.lstatSync(copyFrom).isDirectory()) {
```
Comment (weaverryan) [inline lib/config-generator.js]: I guess we could break these into 2 separate pieces: 1 error is that the path doesn't exist. Another error is that they've pointed directly to a single file, not a directory, right?

File: lib/config-generator.js (hunk 1 above)
Comment (Lyrkan) [reply lib/config-generator.js]: Hmmm, maybe... by "entry" I meant "config entry", would it be better to put that?

File: lib/config-generator.js (hunk 2 above)
Comment (Lyrkan) [reply lib/config-generator.js]: 👍

**Comment by weaverryan** [comment]
Thanks @Lyrkan!

**Pull request merged**